### ISSUE TITLE
Wav2Vec2.0: max_sample_size not working for controlling batch size

### ISSUE THREAD

**Issue by weiyengs** [pytorch/fairseq#3320]
Hi, it seems that using max_sample_size for controlling batch size, and this parameter is advised in this issue ticket: https://github.com/pytorch/fairseq/issues/3262

I'm using a command like this:
python train.py 'examples/wav2vec/train_data_jp' --save-dir 'examples/wav2vec/ftune_models_xlsr_jp' \
--wer-args '("examples/wav2vec/train_data_jp/jp_lm.bin","examples/wav2vec/train_data_jp/lexicon.txt",2,-1)' \
--w2v-path examples/wav2vec/ptrain_models/xlsr_53_56k.pt \
--post-process letter --valid-subset valid --no-epoch-checkpoints --best-checkpoint-metric wer --num-workers 0 \
--max-update 130000 --sentence-avg --task audio_pretraining --arch wav2vec_ctc  \
--labels ltr --apply-mask --mask-selection static --mask-other 0 --mask-length 10 --mask-prob 0.1 --layerdrop 0.05 \
--mask-channel-selection static --mask-channel-other 0 --mask-channel-length 64 --mask-channel-prob 0.1 --zero-infinity \
--feature-grad-mult 0.0 --freeze-finetune-updates 0 --validate-after-updates 10000 --optimizer adam \
--adam-betas '(0.9, 0.98)' --adam-eps 1e-08 --lr 2e-05 --lr-scheduler tri_stage --warmup-steps 1300 --hold-steps 5200 \
--decay-steps 6500 --final-lr-scale 0.05 --final-dropout 0.0 --dropout 0.0 --activation-dropout 0.1 --criterion ctc \
--attention-dropout 0.0 --max-tokens 828000 --seed 2337 --log-format json --log-interval 10 --ddp-backend no_c10d --normalize \
--fp16 --max-sentences 10 --max_sample_size 10

And it output:
train.py: error: unrecognized arguments: --max_sample_size 10

Any ideas? Thanks!

**Comment by harveenchadha** [comment]
it must be --max-sample-size. Can you please check?
